sql-database/failover-groups/add-single-db-to-failover-group-az-cli.sh

```diff
@@ -0,0 +1,112 @@
+﻿#!/bin/bash
+# Set variables
+export subscriptionID=<SubscriptionID>
+export resourceGroupName=myResourceGroup-$RANDOM
+export location=WestUS2
+export adminLogin=azureuser
+export password=`openssl rand -base64 16`
```

Comment:
__Upcoming requirement__: All use of `base64` generation requires that the number of characters be divisible by 3. `openssl rand -base64 18` is recommended.